docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "2.1"
 services:
   db:
-    image: mysql:5.6
+    image: mysql:8.0.28-oracle
```

Comment:
fyi the SRE team recently started cross-compiling mysql5.7, you can use the image named `edxops/mysql:5.7` - so i'd recommend we stick with that unless our production deploy of this app uses 8

https://github.com/edx/devstack/blob/master/docker-compose.yml#L247